B827EBFFFF46EC6D.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFF46EC6D",
+    "servers": [
+      {
+        "server_address": "router.us.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": "-27.571021",
+    "ref_longitude": "-48.508433",
+    "ref_altitude": "",
```

Comment:
this will probably break, use a real altitude or at least, zero (as number, not string).

Reply:
Thank you!